File: src/iniciante/1847/1847.java
import java.io.IOException;
import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) throws IOException {
        int a, b, c;
        Scanner input = new Scanner(System.in);

        a = input.nextInt();
        b = input.nextInt();
        c = input.nextInt();

        if (a > b && c >= b) { System.out.println(":)"); }
        if (a < b && c <= b) { System.out.println(":("); }
        if (a < b && b < c) {
            if ((c - b) < (b - a)) { System.out.println(":("); }
            else { System.out.println(":)"); }
        }
        if (a > b && b > c) {
            if ((b - c) < (a - b)) { System.out.println(":)"); }
            else { System.out.println(":("); }
        }
        if (a == b) {
            if (b < c) { System.out.println(":)"); }
            else { System.out.println(":("); }
        }
    }
}

File: src/iniciante/1847/1847.py
a, b, c = map(int, input().split())

if a > b and c >= b: print(':)')
if a < b and c <= b: print(':(')
if a < b and b < c: print(':(') if (c - b) < (b - a) else print(':)')
if a > b and b > c: print(':)') if (b - c) < (a - b) else print(':(')
if a == b: print(':)') if b < c else print(':(')